### PR TITLE
app-i18n/man-pages-fr: revbump and adoption

### PR DESCRIPTION
**Pull request by beatussum**
- change **URI** because the previous one is down: build from **GitLab** repo
- adopt package: update **metadata.xml**
- change **LICENSE** variable (_sys-apps/man-pages_)

### CODE REVIEW

File: app-i18n/man-pages-fr/files/man-pages-fr-3.70-r1-fix-unpack.patch [hunk 1]
```diff
@@ -0,0 +1,252 @@
+diff --git a/Makefile.old b/Makefile
+index a4e4285..11844b6 100644
+--- a/Makefile.old
++++ b/Makefile
+@@ -8,88 +8,30 @@ WORK_DIR = $(shell pwd)
+ export LANGS
+ export WORK_DIR
+ 
+-unpack: stamp-unpack
```
Comment (mgorny) [inline app-i18n/man-pages-fr/files/man-pages-fr-3.70-r1-fix-unpack.patch]: Don't remove more than necessary, this only makes the patch unnecessarily big. Just modify the dependencies or make calls not to call the targets.

That said, with such intrusive changes to Makefile, I suppose some explanation why you need to do them would be helpful.

File: app-i18n/man-pages-fr/man-pages-fr-3.70-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+S="${WORKDIR}/${PN}-${MY_PV}"
+
+PATCHES=(
+	"${FILESDIR}/${PF}-fix-unpack.patch"
+)
+
+src_unpack() {
+	unpack "${PN}-${MY_PV}-${PR}.tar.gz"
+
+	cd "${S}"
```
Comment (mgorny) [inline app-i18n/man-pages-fr/man-pages-fr-3.70-r1.ebuild]: You should be in `S` already.

File: app-i18n/man-pages-fr/man-pages-fr-3.70-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	unpack "perkamon-man-pages-${PV}.tar.gz"
+	mv "man-pages-${PV}" "perkamon-man-pages-${PV}" || die "the renaming has failed"
```
Comment (mgorny) [inline app-i18n/man-pages-fr/man-pages-fr-3.70-r1.ebuild]: Why do you need to rename it?

File: app-i18n/man-pages-fr/man-pages-fr-3.70-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+	cd "perkamon-man-pages-${PV}"
```
Comment (mgorny) [inline app-i18n/man-pages-fr/man-pages-fr-3.70-r1.ebuild]: All external commands need `|| die`.

File: app-i18n/man-pages-fr/metadata.xml [hunk 5]
```diff
@@ -1,5 +1,32 @@
...
+	</longdescription>
+	<longdescription lang="fr">
+		Le projet perkamon a pour objectif de faciliter la traduction des pages de  manuel
+		Linux en convertissant, à l'aide de l'outil <pkg>app-text/po4a</pkg>, leur contenu
+		en fichiers PO. Ces fichiers PO sont identiques à ceux utilisés habituellement par
+		les traducteurs. La conversion est effectuée dans les deux sens par le responsable
+		de la traduction, les traducteurs ne s'occupent que de traduire les fichiers PO.
+	</longdescription>
+	<upstream>
+		<maintainer status="active">
```
Comment (mgorny) [inline app-i18n/man-pages-fr/metadata.xml]: Honestly? I don't think this is actually helpful information.

**Comment by beatussum** [comment]
@mgorny I deleted the unnecessary information in _metadata.xml_ and created a new and easier patch.

File: app-i18n/man-pages-fr/files/man-pages-fr-3.70-r1-fix-unpack.patch [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
++++ b/Makefile
+@@ -39,9 +39,9 @@ stamp-unpack-git:
+ 	touch $@
+ 
+ setup: stamp-setup
+-stamp-setup: stamp-unpack
+-	$(MAKE) -C perkamon setup
+-	mv perkamon/build .
++stamp-setup:
++	$(MAKE) -C man-pages-$(PERKVERSION) setup
```
Comment (mgorny) [inline app-i18n/man-pages-fr/files/man-pages-fr-3.70-r1-fix-unpack.patch]: But why do you need to change it? Wouldn't it be simpler to just rename the directory after unpacking?

File: app-i18n/man-pages-fr/files/man-pages-fr-3.70-r1-fix-unpack.patch [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+-stamp-setup: stamp-unpack
++stamp-setup:
+ 	-rm -rf build
+ 	mkdir -p build/C
+ 	for i in $$(seq 8); do mkdir build/C/man$$i; done
+@@ -109,12 +109,12 @@ stamp-setup: stamp-unpack
+ 	#      target_man_page src_man_page
+ 	#  This file is used after pages are translated to create aliases
+ 	#  of translated manual pages.
+-	set -e; for f in man-pages/man?/*.?; do \
```
Comment (mgorny) [inline app-i18n/man-pages-fr/files/man-pages-fr-3.70-r1-fix-unpack.patch]: Same: rename it?

**Comment by beatussum** [comment]
@mgorny I have added both `mv` and update the patch file as requested.

**Pull request closed without merging**